src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -195,8 +219,10 @@ private static string GetLinuxName()
                         }
                     }
                 }
-            }
 
+                throw new Exception($"Can't find VERSION_ID in {filePath}");
```

Comment:
We return everywhere in this tool. So, if we want raise exceptions, we probably should do so everywhere else 😃

Reply:
added exception for the similar methods for other platforms